pkg/graph/schema.resolvers.go

```diff
@@ -2019,34 +2020,23 @@ func (r *tRBAdminNoteResolver) CategorySpecificData(ctx context.Context, obj *mo
 }
 
 // Author is the resolver for the author field.
-func (r *tRBAdviceLetterResolver) Author(ctx context.Context, obj *models.TRBAdviceLetter) (*models.UserInfo, error) {
-	authorInfo, err := dataloaders.FetchUserInfoByEUAUserID(ctx, obj.CreatedBy)
-	if err != nil {
-		return nil, err
-	}
-
-	return authorInfo, nil
+func (r *tRBGuidanceLetterResolver) Author(ctx context.Context, obj *models.TRBGuidanceLetter) (*models.UserInfo, error) {
+	panic(fmt.Errorf("not implemented: Author - author"))
```

Comment:
whoops?

Reply:
Yeah, this can probably just be copied from the old resolver on the same line (calling `dataloaders.FetchUserInfoByEUAUserID`)

Reply:
whoops! nice catch Lee, thank you for catching and fixing